frontend/app/src/people/utils/__tests__/filterValidation.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { bountyHeaderFilter, bountyHeaderLanguageFilter } from '../filterValidation';
+
+describe('testing helpers', () => {
+  describe('bountyHeaderFilter', () => {
+    test('o/t/t', () => {
+      expect(bountyHeaderFilter({ Open: true }, true, true)).toEqual(false);
+    });
+    test('a/t/t', () => {
+      expect(bountyHeaderFilter({ Assigned: true }, true, true)).toEqual(true);
+    });
+    test('p/t/t', () => {
+      expect(bountyHeaderFilter({ Paid: true }, true, true)).toEqual(true);
+    });
+    test('/t/t', () => {
+      expect(bountyHeaderFilter({}, true, true)).toEqual(true);
+    });
+    test('o/f/t', () => {
+      expect(bountyHeaderFilter({ Open: true }, false, true)).toEqual(false);
+    });
+    test('a/f/t', () => {
+      expect(bountyHeaderFilter({ Assigned: true }, false, true)).toEqual(true);
+    });
+    test('p/f/t', () => {
+      expect(bountyHeaderFilter({ Paid: true }, false, true)).toEqual(false);
+    });
+  });
+  describe('bountyHeaderLanguageFilter', () => {
+    test('match', () => {
+      expect(bountyHeaderLanguageFilter(['Javascript', 'Python'], { Javascript: true })).toEqual(
+        true
+      );
+    });
```

Comment:
looks like this should be `false`

Reply:
Why is that? Do struct values not compare with strings? This could be highlighting a problem I've been facing where none of the language filters work for me.